### PR TITLE
Use file based signer now that Rekor supports it.

### PR DESCRIPTION
**Pull request by vaikas**
Signed-off-by: Ville Aikas <vaikas@chainguard.dev>

#### Summary
Now that Rekor supports file based signing key, use that so any Rekor restart won't screw everything up by creating a new key.
This was added here:
https://github.com/sigstore/rekor/pull/1049
Since there hasn't been a Rekor release since, we choose the one that was created by CI build to get the tests going using this.

#### Release Note
Use file based signing key for Rekor.

#### Documentation
Updated the README.

### CODE REVIEW

File: config/rekor/rekor/300-rekor.yaml [hunk 1]
```diff
@@ -19,7 +19,11 @@ spec:
       serviceAccountName: rekor
       containers:
       - name: rekor
-        image: gcr.io/projectsigstore/rekor-server@sha256:af91aee88c6d8efb3175b441420a4c61691ec5d6a72303c8ef47750c1dfd2325 # v0.11.0
+        # This has been built from here to pick up the file based signer change:
+        # https://github.com/sigstore/rekor/actions/runs/3061668501/jobs/4941722492
+        # Which contains this change that we need.
+        # https://github.com/sigstore/rekor/pull/1049
+        image: gcr.io/projectsigstore/rekor/ci/rekor/rekor-server@sha256:91a6fa93804e1c0b48942a6a0e38b30b851a9f281a65e84a05cd85c138450d7f
```
Comment (k4leung4) [inline config/rekor/rekor/300-rekor.yaml:26]: todo to switch back to gcr.io/projectsigstore/rekor-server with the next release?

**Review by k4leung4 — DISMISSED** [review]

File: config/rekor/rekor/300-rekor.yaml (hunk 1 above)
Comment (vaikas) [reply config/rekor/rekor/300-rekor.yaml:26]: Yup, great call!
https://github.com/sigstore/scaffolding/issues/359

**Review by k4leung4 — DISMISSED** [review]

**Comment by vaikas** [comment]
Hrm, did I bring the wrong worsion of rekor 😭 
```
  2022-09-16T04:05:09.065Z  ERROR  app/root.go:53  unknown flag: --rekor_server.signer-pwd
  github.com/sigstore/rekor/cmd/rekor-server/app.Execute
    github.com/sigstore/rekor/cmd/rekor-server/app/root.go:53
  main.main
    github.com/sigstore/rekor/cmd/rekor-server/main.go:21
  runtime.main
    runtime/proc.go:250
```

**Comment by vaikas** [comment]
I might have, but I also specified the wrong freaking flag name. Getting late 🤣

**Review by k4leung4 — APPROVED** [review]

**Pull request merged**